packages/fxa-settings/src/lib/channels/helpers.ts

```diff
@@ -2,93 +2,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { AccountData } from '../../models';
+import firefox, { FxALoginRequest } from './firefox';
 
-// const ALLOWED_LOGIN_FIELDS = [
-//   'declinedSyncEngines',
-//   'email',
-//   'keyFetchToken',
-//   'offeredSyncEngines',
-//   'sessionToken',
-//   'services',
-//   'uid',
-//   'unwrapBKey',
-//   'verified',
-// ];
-
-// const REQUIRED_LOGIN_FIELDS = [
-//   'email',
-//   'keyFetchToken',
-//   'sessionToken',
-//   'uid',
-//   'unwrapBKey',
-//   'verified',
-// ];
-
-// Might need to go on the Integration?
+// TODO: might need to go on Integration?
 // let uidOfLoginNotification: hexstring = '';
 
-// function hasRequiredLoginFields(loginData) {
-//   const loginFields = Object.keys(loginData);
-//   return (
-//     REQUIRED_LOGIN_FIELDS.filter((field) => !loginFields.includes(field))
-//       .length === 0
-//   );
-// }
-
-/**
- * Get login data from `account` to send to the browser.
- * All returned keys have a defined value.
- */
-// function getLoginData(account: AccountData) {
-// let loginData: Partial<AccountData> = {};
-// for (const key of ALLOWED_LOGIN_FIELDS) {
-//   loginData[key] = account[key];
-// }
-//   // TODO: account for multiservice when we combine reliers
-//   // const isMultiService = this.relier && this.relier.get('multiService');
-//   // if (isMultiService) {
-//   //   loginData = this._formatForMultiServiceBrowser(loginData);
-//   // }
-//   loginData.verified = !!loginData.verified;
-//   // TODO: this is set in the `beforeSignIn` auth-broker method
-//   // loginData.verifiedCanLinkAccount = !!this._verifiedCanLinkEmail;
-//   return Object.fromEntries(
-//     Object.entries(loginData).filter(([key, value]) => value !== undefined)
-//   );
-// }
+const REQUIRED_LOGIN_FIELDS: Array<keyof FxALoginRequest> = [
```

Comment:
There's a link to [what params](https://searchfox.org/mozilla-central/rev/82828dba9e290914eddd294a0871533875b3a0b5/services/fxaccounts/FxAccounts.sys.mjs#910) are used on the FF side above `FxALoginRequest`. This is different than the previous "allowed" and "required" fields which were commented out, please [see what we're doing in this file](https://github.com/mozilla/fxa/blob/d022d066c4431d219c63050397376c78a4f4d17e/packages/fxa-content-server/app/scripts/models/auth_brokers/fx-sync-channel.js#L20-L21) and double check me. 😎

Reply:
Looks right. One comment, is that although `authAt` is listed in the FF source as a field, but I don't see it being used, and it looks like the content-server auth broker model doesn't have it listed as a required field. It seems like what you have is technically more correct though, so it is probably fine.